src/test/java/com/scalyr/integrations/kafka/ScalyrSinkTaskTest.java

```diff
@@ -507,6 +508,7 @@ public void testFlush() {
   /**
    * Verify stop doesn't throw any exceptions
    */
+  @Ignore("Test closes AddEventsClient AsyncHttpClient, which causes subsequent tests to fail")
```

Comment:
Ok, are there problems with us not being able to call stop?

Reply:
I thought about this some more.  The `AsyncHttpClient` life cycle should match the lifecycle of the connector.  When the task stops, we should not close the `AsyncHttpClient`.  I changed this to have the `AsyncHttpClient` match the lifecycle for the connector.